src/components/List/ListItem/ListItem.jsx

```diff
@@ -116,11 +121,22 @@ const ListItem = ({
 }) => {
   const handleExpansionClick = () => isExpandable && onExpand(id);
 
+  if (__DEV__ && Array.isArray(rowActions)) {
+    warning(
+      false,
+      'You have passed an array of nodes to ListItem as rowActions.  This can cause performance problems and has been deprecated.  You should pass a render function instead.'
+    );
+  }
+
   const renderNestingOffset = () => {
     return nestingLevel > 0 ? (
       <div
         className={`${iotPrefix}--list-item--nesting-offset`}
-        style={{ width: `${nestingLevel * 30}px` }}
+        style={{
+          width: `${
+            nestingLevel === 1 && !isExpandable ? 30 : (nestingLevel - 1) * 30
```

Comment:
This is still regressed. `Request 2` should be indented and is not.

![image](https://user-images.githubusercontent.com/3360588/99303809-2086dc00-2817-11eb-9711-9a9837a034a6.png)

What's wrong with the current implementation? The spec calls for each list item to reserve the space for the chevron icon when the list has a hierarchy.